tests/lib/netinfo_test.py

```diff
@@ -168,8 +168,6 @@ def test_warning_show(nm_mock, nm_dns_mock):
     nm_dns_mock.get_running.return_value = current_config[DNS.KEY][DNS.RUNNING]
     nm_dns_mock.get_config.return_value = current_config[DNS.KEY][DNS.CONFIG]
 
-    with pytest.warns(FutureWarning) as record:
+    with pytest.raises(TypeError):
+        # pylint: disable=E1121
```

Comment:
Use proper name instead of E1121.

Please re-enable the pylint after the line you want  to disable.